packages/tasks/src/local-apps.ts

```diff
@@ -82,6 +82,10 @@ function isLlamaCppGgufModel(model: ModelData) {
 	return !!model.gguf?.context_length;
 }
 
+function isMlxModel(model: ModelData) {
+	return model.tags.includes("mlx");
```

Comment:
```suggestion
	return model.library_name === "mlx";
```

or maybe like this @Vaibhavs10 if we want to be pedantic about it

Reply:
actually nvm because https://huggingface.co/mlx-community/Llama-3.2-3B-Instruct-4bit for instance is not library=mlx

Reply:
Yes! Just `mlx` tag would be more reliable 👍